fedimint-client-wasm/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+edition = "2021"
+name = "fedimint-client-wasm"
+version.workspace = true
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+anyhow.workspace = true
+async-stream = "0.3.5"
+fedimint-core.workspace = true
+fedimint-ln-client = { workspace = true, features = ["cli"] }
+fedimint-mint-client = { version = "=0.5.0-alpha", path = "../modules/fedimint-mint-client" }
+fedimint-client = { workspace = true }
+wasm-bindgen = "=0.2.92"                                                                      # must match the nix provided wasm-bindgen-cli version
+wasm-bindgen-futures = "0.4.42"
+wasm-bindgen-test = "0.3.42"
+fedimint-api-client.workspace = true
+serde_json.workspace = true
```

Comment:
:exploding_head:

Reply:
you didn't know about this toml syntax?

Reply:
oh this broke deps sorting :smile:

Reply:
Just because I know about the syntax, doesn't mean I'm clever enough to succinctly use it every time. :D